WalletWasabi.Gui/Controls/WalletExplorer/ClosedWalletViewModel.cs

```diff
@@ -0,0 +1,73 @@
+using AvalonStudio.Extensibility;
+using ReactiveUI;
+using Splat;
+using System;
+using System.IO;
+using System.Linq;
+using System.Reactive;
+using System.Reactive.Linq;
+using System.Threading;
+using WalletWasabi.Gui.Helpers;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.Gui.Controls.WalletExplorer
+{
+	public class ClosedWalletViewModel : WalletViewModelBase
+	{
+		public ClosedWalletViewModel(string path) : base(Path.GetFileNameWithoutExtension(path))
+		{
+			OpenWalletCommand = ReactiveCommand.CreateFromTask(async () =>
+			{
+				try
+				{
+					IsBusy = true;
+
+					var global = Locator.Current.GetService<Global>();
+
+					if (!await global.WaitForInitializationCompletedAsync(CancellationToken.None))
+					{
+						return;
+					}
+
+					var walletManager = global.WalletManager;
+
+					var keyManager = global.LoadKeyManager(Title);
```

Comment:
All the wallets were loaded before and WalletManager has them.

Reply:
+

Reply:
done